src/commands/functions/create.js

```diff
@@ -346,36 +346,33 @@ function installAddons(addons = [], fnPath) {
     }
     this.log(`${NETLIFYDEVLOG} checking Netlify APIs...`)
 
-    return api.getSite({ siteId }).then(siteData => {
-      const accessToken = api.accessToken
-      const arr = addons.map(({ addonName, addonDidInstall }) => {
-        this.log(`${NETLIFYDEVLOG} installing addon: ` + chalk.yellow.inverse(addonName))
-        // will prompt for configs if not supplied - we do not yet allow for addon configs supplied by `netlify functions:create` command and may never do so
-        return createSiteAddon(accessToken, addonName, siteId, siteData, this.log)
-          .then(async addonCreateMsg => {
-            if (addonCreateMsg) {
-              // spinner.success("installed addon: " + addonName);
-              if (addonDidInstall) {
-                const { addEnvVariables } = require('../../utils/dev')
-                await addEnvVariables(api, site)
-                const { confirmPostInstall } = await inquirer.prompt([
-                  {
-                    type: 'confirm',
-                    name: 'confirmPostInstall',
-                    message: `This template has an optional setup script that runs after addon install. This can be helpful for first time users to try out templates. Run the script?`,
-                    default: false,
-                  },
-                ])
-                if (confirmPostInstall) addonDidInstall(fnPath)
-              }
+    const siteData = await api.getSite({ siteId })
+    const arr = addons.map(async ({ addonName, addonDidInstall }) => {
+      this.log(`${NETLIFYDEVLOG} installing addon: ` + chalk.yellow.inverse(addonName))
+      try {
+        const addonCreated = await createSiteAddon(api.accessToken, addonName, siteId, siteData, this.log)
+        if (addonCreated) {
+          if (addonDidInstall) {
+            const { addEnvVariables } = require('../../utils/dev')
```

Comment:
Do we have a good reason to dynamically load `require()` here (I think not)?

Reply:
Definitely not